TRANSLATION.md

```diff
@@ -1,58 +1,61 @@
-as long as they are attibuted properly.  I am happy to help those
-adopting/adapting these materials make sure their acknowledgement is
-sufficient.  I am always excited to learn of new translations and promote them
-to my students.
-
-All the material for this course (web site, autograders, PowerPoint, textbook, 
-figures for the textbook, assignments, etc) is in this GitHub repository.  The
-only material that is not in github are the videos and I can provide those as 
-well if they need to be hosted behind a firewall or in a country that has banned 
-YouTube.
-
-You could also provide translations on the YouTube channel 
+Traduciendo Python para Todos
+=============================
+
+Los materiales del curso están licenciados bajo Creative Commons, por lo que pueden usarse
+siempre que se haga la correspondiente atribución. Estaré encantado de ayudar a
+quienes usen o quieran usar estos materiales a comprobar que su atribución
+es suficiente. Siempre recibo gustoso la noticia de nuevas traducciones, las que
+promociono a mis estudiantes.
+
+Todo el material de este curso (sitio web, autoevaluadores, presentaciones PowerPoint,
+libros de texto, imágenes del mismo, tareas, etc) se encuentra en este repositorio de GitHub. El
+único material que no se encuentra en GitHub son los videos, aunque puedo enviar
+copias si se necesita alojarlos detrás de un firewall o en un país que haya
+limitado el acceso a Youtube.
+
+También pueden hacer traducciones en nuestro canal de Youtube
 https://www.youtube.com/playlist?list=PLlRFEj9H3Oj7Bp8-DfGpfAfDBiblRfl5p
 
-My ultimate goal would a series of complete web sites at urls like es.py4e.com, 
-cn.py4e.com, ir.pye4.com with completely translated user experiences that also function
-as OER repositories for Python materials in those languages.  Of course you do not
-need to use my urls, but I do want to promote the translations of this material
-and am happy to point my subdomains at your servers or forward requests at those
-subdomains to your servers.
-
-I have set this up so you can fork my github repository and begin the translation
-work. I will be happy to help you get your translated work up and available.  I 
-have an automatic build server for the text book that takes your latest book in 
-github and produces the PDF and EPUB and hosts it online at 
+Mi meta final sería una serie de sitios web completos en urls como es.py4e.com, 
+cn.py4e.com o ir.pye4.com con experiencias de usuario completamente traducidas que
+además funcionen como repositorios OER para los materiales de Python en esos lenguajes.
+Evidentemente, no necesitas usar mis urls, pero quiero promover la traducción de estos 
+materiales y estaré encantado de apuntar mis subdominios a tus servidores o reenviar
+solicitudes a estos subdominios a tus servidores.
+
+He configurado mi repositorio de github de forma tal que puedas hacer una 
+bifurcación de éste y comenzar a traducirlo. Estaré encantado de subir y poner
```

Comment:
I am not sure how other Spanish speakers talk about a fork. In the case of Mexico, the term "fork" is used as is, but I am not sure about other countries. @jdougnac have you heard of the translation of the word in another countries?

Reply:
Maybe it would be clear if we say something like "...tal que puedas hacer una copia (fork) de éste y comenzar...".

Reply:
efectuada la modificación